plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -231,6 +230,30 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
     this._tracerProvider = tracerProvider;
   }
 
+  private _getForceFlushFn() {
+    if (!this._tracerProvider) {
+      diag.error(
+        'Spans are not exported for the lambda function as the tracerProvider is undefined.'
+      );
+      return undefined;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let currentProvider: any = this._tracerProvider;
+
+    if (typeof currentProvider.getDelegate === 'function') {
+      currentProvider = currentProvider.getDelegate();
+    }
+
+    if (typeof currentProvider.getActiveSpanProcessor === 'function') {
+      const activeSpanProcessor = currentProvider.getActiveSpanProcessor();
+      if (typeof activeSpanProcessor.forceFlush === 'function')
+        return activeSpanProcessor.forceFlush;
+    }
```

Comment:
```suggestion
    if (typeof currentProvider.getActiveSpanProcessor === 'function') {
      return currentProvider.getActiveSpanProcessor();
    }
```

Only get the span processor from this helper function.